nightwatch/test.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+ENV=$1
+[ -z "$ENV" ] && ENV='default'
+
+if [ "$ENV" != "default" ] ; then
+    CRED="../mycredentials"
+    [ ! -f $CRED ] && echo "Could not find credentials file: $CRED" && exit 1
+    CRED="env `cat $CRED`"
+fi
+
+$CRED ./node_modules/.bin/nightwatch --env $ENV
```

Comment:
please just wrap nightwatch so make this:
```shell
$CRED ./node_modules/.bin/nightwatch $@
```
which will allow more versatile use, e.g. adding tags to the params etc